tools/cli/go-whisk/whisk/client.go

```diff
@@ -126,6 +127,10 @@ func NewClient(httpClient *http.Client, config *Config) (*Client, error) {
         config.Version = "v1"
     }
 
+    if len(config.UserAgent) == 0 || config.UserAgent == "/not set" {
```

Comment:
Where does `/not set` come from?

Reply:
the CLI version, In my testing I saw the plugin will show `OW-Plugin/not set`, so I added it here to cover the missing UserAgent case, aka "/not set"

Reply:
Do you think that is because the version isn't being set in that scenario? @mdeuser, see the above.

Reply:
the plugin doesn't use the same gradle build I believe (or maybe I didn't build it correctly), so yes, it is not being set. I was thinking I can check for this prior to appending the cli version in commands.go if we wanna move it.

Reply:
from main.go
```
// CLI_BUILD_TIME holds the time of the CLI build.  During gradle builds,
// this value will be overwritten via the command:
//     go build -ldflags "-X main.CLI_BUILD_TIME=nnnnn"   // nnnnn is the new timestamp
var CLI_BUILD_TIME string = "not set"
```

Reply:
Might be easier to just set the version in the plugin build.

Reply:
yep.. it already is.  the "not set" CLI version is a result of a local build i.e. `go build main.go` without providing the CLI_BUILD_TIME linker flag

Reply:
oops. sorry,  as soon as i saw the "not set" i was thinking this was in reference to the cli version...

Reply:
sooo.. i don't believe the user-agent var will every have a "not set" value.